### DISCUSSION TITLE
a limit of active loaders per player

### DISCUSSION THREAD

**Discussion by condblock** [BG-Software-LLC/WildLoaders#55] [Approved Feature Requests]
I think use multiple loaders is very over-powered on some servers. Example, to give the player a permission like wildloaders.limit.(loader name).(number) or limit:(number) on config to set the limit.

**Comment by ChrissW-R1** [comment]
This is a feature we really need. If every player could use unlimited loaders, we need a server, which have the power of 5 or 6 times the player limit is.
